src/doc/dev_manual/CreateRelease.rst

```diff
@@ -4,55 +4,136 @@ Creating a Release
 Overview
 --------
 
-When we put out a new release we should tag the repository and create a
-release. We will describe creating a release by way of example using the
-steps used to create the 3.0.1 release.
+When we put out a new release we create two initial assets and tag the release candidate branch.
 
-Tagging the release
--------------------
+1) A source code tar file that includes a ``build_visit`` script that references the tagged release.
+2) A unified ``build_visit`` script that references the tagged release and includes the checksum for the source code tar file.
 
-To create a release you will first create a tag using git commands. You
-should get the short SHA for the release that can be found on the splash
-screen of any of the binaries built from the source tar file. The Linux
-distributions are all built with the source tar file. The Windows
-distribution is typically not. To bring up the splash screen go to
-*Help->About*.
+Creating the release
+--------------------
 
-.. figure:: images/Release-SplashScreen.png
+We will describe creating a release by way of example using the steps used to create the 3.3.3 release.
 
-   The splash screen with the short SHA.
+Commit a change that references the tagged release, which in our case is 3.3.3. ::
 
-Now you can issue the git commands to create the tag and push it to GitHub. ::
+    git checkout 3.3RC
+    git pull
+    git checkout -b task/brugger1/2023_03_30_build_visit
+    vi src/tools/dev/scripts/build_visit
+    git add src/tools/dev/scripts/build_visit
+    git commit -m "Temporarily set release in build_visit to v3.3.3."
+    git push --set-upstream origin task/brugger1/2023_03_30_build_visit
+    git branch -D task/brugger1/2023_03_30_build_visit
 
-    git checkout 3.0RC
-    git checkout 2f38385
-    git tag v3.0.1
-    git push origin v3.0.1
+At this point you are ready to tag the release. ::
 
-If you go to GitHub and go to the *Releases* tab you will see the newly
-created tag. Now you are ready to create the release. Click on
-*Draft a new release* to bring up the form to create a new release. 
+    git checkout 3.3RC
+    git pull
+    git tag v3.3.3
+    git push origin v3.3.3
+
```

Comment:
```suggestion

Creating first two release assets: source tarball and build_visit script
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
FYI...it looks like GitHub's **Suggest change** feature does not use fixed width fonts and so in cases like this, I've made an error in making heading underline match number of chars in heading.